containers/python/pywrencloudpickle.py

```diff
@@ -0,0 +1,878 @@
+"""
```

Comment:
Pulled straight from Pywren, with the exception of some formatting changes from calling the formatting script. Is there an appropriate way to attribute it / deal with files taken from other repos like this one?